pennylane/ops/op_math/controlled.py

```diff
@@ -127,6 +127,8 @@ def cond_fn():
         available_eps = compiler.AvailableCompilers.names_entrypoints
         ops_loader = available_eps[active_jit]["ops"].load()
         return ops_loader.ctrl(op, control, control_values=control_values, work_wires=work_wires)
+    if qml.math.is_abstract(op):
+        return Controlled(op, control, control_values=control_values, work_wires=work_wires)
```

Comment:
This does not seem to work if there are multiple control wires, we'll need to handle them like `Operator.wires` I think.

```python
def qfunc(op):
    qml.ctrl(op, control=[3, 4], control_values=[0, 1])

jaxpr = jax.make_jaxpr(qfunc)(qml.IsingXX(1.2, wires=(0, 1)))
>>> TypeError: Value [3, 4] with type <class 'list'> is not a valid JAX type
```

Reply:
good catch!